holoviews/core/spaces.py

```diff
@@ -495,6 +495,13 @@ def __call__(self, *args, **kwargs):
         return ret
 
 
+class OverlayCallable(Callable):
+    """
+    A Callable subclass specifically meant to indicate that the Callable
+    represents an overlay operation between two objects.
+    """
```

Comment:
Ok, the diff didn't make it clear that this class has methods too. But why can't those methods be on ``Callable``?